_search-plugins/sql/settings.md

```diff
@@ -78,3 +78,37 @@ Setting | Default | Description
 `plugins.sql.cursor.keep_alive` | 1 minute | Configures how long the cursor context is kept open. Cursor contexts are resource-intensive, so we recommend a low value.
 `plugins.query.memory_limit` | 85% | Configures the heap memory usage limit for the circuit breaker of the query engine.
 `plugins.query.size_limit` | 200 | Sets the default size of index that the query engine fetches from OpenSearch.
+
+## Spark connector settings
+
+The SQL plugin supports [Apache Spark](https://spark.apache.org/) as a data source. This allows you to run SQL queries in Spark and seamlessly write the resulting data back to OpenSearch. To use Spark, enable the following settings to add Spark as a data source and enable the correct permissions.
```

Comment:
The SQL plugin supports Apache Spark as an augmented compute source. Data sources defined in Apache Spark as tables will be consumable from OpenSearch. OpenSearch users will be able to execute SQL queries against external sources from within OpenSearch Dashboards Discover and Observability Logs. To get started, enable the following settings to add Spark as a data source and enable the correct permissions.

Reply:
Switched it to this: 

---

The SQL plugin supports [Apache Spark](https://spark.apache.org/) as an augmented compute source. When data sources are defined as tables in Apache Spark, OpenSearch can consume those tables. This allows you to run SQL queries against external sources inside OpenSearch Dashboard's [Discover]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/) and observability logs. 

To get started, enable the following settings to add Spark as a data source and enable the correct permissions.

Reply:
That works.